databases/week2/MoreQueries.sql

```diff
@@ -0,0 +1,36 @@
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT user.id,task.title, user.email
+FROM user, user_task, task WHERE
+user.email LIKE "%@spotify.com" AND
+user.id=user_task.user_id AND
+user_task.task_id=task.id;
```

Comment:
Does table **user_task** exist in the given design of the DB queris?
try this if it  gives any result out

```
SELECT user.id,title, email
FROM task
  JOIN      user
on task.id=user.id
WHERE user.email LIKE "%@spotify.com";